recipe/meta.yaml

```diff
@@ -13,10 +13,13 @@ source:
 
 build:
   skip: true  # [win]
-  number: 0
+  number: 1
 
 outputs:
   - name: {{ name }}
+    build:
+      run_exports:
+        - {{ pin_subpackage('nceplibs-g2c', max_pin='x.x.x') }}
```

Comment:
@conda-forge/g2clib, do any of you know what the ABI stability for nceplibs-g2c is like?  Do we need to pin run-exports to the patch version or can we get away with something looser?

Reply:
Hi @xylar.  Good question and I am not sure 100%.  I am a contributor to NCEPLIBS-g2c.  I know that the v2 of nceplibs-g2c introduced a new API paradigm while retaining v1.  It is likely that v1 functionality could be deprecated and removed at some point.

You might want to reach out to https://github.com/edwardhartnett who is the code manager.

Reply:
@edwardhartnett, do you, by chance, have advice for us on this?